iOS/MSCoreKit/Sources/MSNetworking/Router.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Router.swift
+//
+//
+//  Created by 전민건 on 11/16/23.
+//
+
+import Foundation
+
+struct RouterType {
+    
+    private var encodable: Encodable?
+    
+    //기능별 Router
+    var getJourney: Router {
+        Router(baseURL: .none, pathURL: .none, method: .get, body: HTTPBody(content: encodable))
+    }
+    var getPerson: Router {
+        Router(baseURL: .none, pathURL: .none, method: .get, body: HTTPBody(content: encodable))
+    }
+    
+}
+
+struct Router: Requestable {
```

Comment:
```suggestion
struct MSRouter: Router {
```

뭔가 네이밍이 어색한 것 같은데 이건 어떠신가요? 👀

Reply:
이런 피드백 너무 좋습니다. 감사합니다!